jetty-gcloud/jetty-gcloud-session-manager/src/main/java/org/eclipse/jetty/gcloud/session/GCloudSessionDataStore.java

```diff
@@ -65,7 +66,7 @@
     protected boolean _indexesPresent = false;
     protected EntityDataModel _model;
     protected boolean _modelProvided;
-    private String _namespace;
+    private String _namespace = DEFAULT_NAMESPACE;
```

Comment:
This is an embedded mode change, it used to return null here, now it returns empty string.